pkg/mapper/configmap/configmap.go

```diff
@@ -21,6 +21,13 @@ import (
 	"sigs.k8s.io/aws-iam-authenticator/pkg/config"
 )
 
+const (
+	metricSuccess     = "success"
```

Comment:
Seems weird to define these in the "configmap" package, since they look pretty generic.  They probably belong in a metrics package which we can create now.

Reply:
can we take care of this